src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -2135,6 +2135,24 @@ def load_arguments(self, _):
             "revision", validator=validate_azure_service_mesh_revision, required=True
         )
 
+    with self.argument_context("aks mesh upgrade rollback") as c:
```

Comment:
```suggestion
    with self.argument_context("aks mesh upgrade") as c:
```
could merge these 2 blocks, cli framework will match the sub commands with the same prefix

Reply:
we do have `az aks mesh upgrade start` command as well which doesn't define `--yes` as part of it.